### DISCUSSION TITLE
QuestDB driver for Dbeaver

### DISCUSSION THREAD

**Discussion by rodryquintero** [questdb/questdb#4363] [Ideas]
I have tried using the native Postgresql drivers, but they throw errors related to tables that are created by default on Postgres databases but are missing in QuestDB. Dbeaver is a very popular RDBMS gui tool. It will help adoption of questdb to have a native driver for this application.

https://dbeaver.io/

**Comment by Hailios** [comment]
any update here? does DBeaver work with QuestDB nowadays?

**Reply by nwoolmer** [reply]
We have rewritten our Postgres connector, but haven't spent time exploring DBeaver. Would you be able to try it out and let us know if you hit any trouble?

We do work with more offerings now, for example PowerBI and RStudio.

**Reply by Hailios** [reply]
the data views, with queries, seem to work now. it cannot view the table metadata like columns etc, but being able to query the data is enough for my usecase. thanks!

**Reply by nwoolmer** [reply]
Great news!

Indeed, we support only a subset of metadata functions, so schema introspection often doesn't work perfectly.

We hope to improve compatibility further in future with a Quest-native driver, but until then, at least you can run queries!
